WcaOnRails/app/models/user_group.rb

```diff
@@ -90,6 +91,10 @@ def lead_user
     end
   end
 
+  def child_groups
+    UserGroup.where(parent_group_id: self.id)
+  end
```

Comment:
This looks like it can be implemented through a `has_many` relation.
See `app/models/stripe_transaction.rb` around line 30-ish for inspiration.

Reply:
(This is important because it's not only shorter, but also much more efficient when Rails computes SQL queries)

Reply:
Done, this is nice :-)